Introduced SmartCash cronjobs to improve stability of the node

lapistano opened the pull request.

Commit:
added monitoring scripts to be added as cronjobs

Commit:
added some monitoring cronjobs

Commit:
added script to install and open firewall for 22/tcp and 9678

lapistano committed:
introduced SmartCash cronjobs to ensure deamon stability

tunikum commented on Dockerfile:
```diff
@@ -5,14 +5,17 @@ ARG _smartcashBin=/opt/smartcash/smartcashd
+ENV WALLET_RUN /run/smartcash
 
 RUN apt-get update && \
+    apt-get upgrade -y  && \
```
`apt-get upgrade` is bad practice according to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

lapistano replied:
I c .. kk will change

tunikum commented on Dockerfile:
```diff
@@ -21,10 +24,14 @@ RUN mkdir -p `dirname $WALLET_CONF` && \
-VOLUME /data
+RUN mkdir -p `dirname $WALLET_RUN`
+
+ADD /cronjobs.tgz  $WALLET_RUN
```
Does it have to be a `tgz`? Shouldn't `COPY bin $WALLET_RUN` do it? Then it wouldn't be necessary to check in a binary file.

tunikum commented on openfw.sh:
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
This file seems to be unnecessary.

tunikum commented on Dockerfile:
```diff
@@ -21,10 +24,14 @@ RUN mkdir -p `dirname $WALLET_CONF` && \
     ln -s $_entryPointBin /usr/local/bin/docker-entry
 
-VOLUME /data
+RUN mkdir -p `dirname $WALLET_RUN`
```
This shouldn't be necessary.

lapistano committed:
removed dependency on the zip file

# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# Date:      Tue Dec 11 23:01:58 2018 +0100
#
# On branch master
# Your branch is ahead of 'origin/master' by 1 commit.
#   (use "git push" to publish your local commits)
#
# Changes to be committed:
#	modified:   Dockerfile
#	deleted:    cronjobs.tgz
#

lapistano committed:
removed fw script

lapistano committed:
rmoved reference to nano
